00-HTML-CSS-basics/Proyecto2/style.css

```diff
@@ -0,0 +1,51 @@
+body {
+  /*font-family:Arial, Helvetica, sans-serif;*/
+  font-size: 14px;
+}
+
+.container  {
+  width: 960px;
+  /* TOP RIGHT BOTTOM LEFT*/
+  margin: 0 auto 0 auto;
+  position: relative;
+
+}
+
+.header {
+  position: center;
+  background-color: magenta;
+  text-align: center;
+  font-size: 46px;
+}
+.navigation {
+  float:left;
+  background-color: blue;
+  display: block;
+  font-size: 12px;
+}
+.content {
+  width: 800px;
+  float:center;
+  text-align: center;
+  font-size: 14px;
+  background-color: blue;
+}
+.articulo {
+  display: block;
+
+}
+.sidebar  {
+  float: right;
+  width: 100px;
+  height: 170px;
+  background-color: magenta;
+  font-size: 10px;
+}
+.footer {
+  width: 960px;
+  margin: 100px auto 0 auto;
+  text-align: center;
+  background-color: magenta;
+  position: relative;
+  font-size: 10px;
+}
```

Comment:
What about point 4-7 of the Specificity topic? It would be nice one of them applied :)